### PR TITLE
Add Zathura package, its Poppler-PDF plugin and display system

### PR DESCRIPTION
**Pull request by vincowl**

### CODE REVIEW

File: packages/girara.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  source_url 'https://git.pwmt.org/pwmt/girara/-/archive/0.3.4/girara-0.3.4.tar.bz2'
+  source_sha256 'e78257e4218a0f7f59cc1bea472c7c6794fa51cd9261d87affbe731c1e22c6a5'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  depends_on 'meson' => :build
+  depends_on 'gettext' => :build
```
Comment (uberhacker) [inline packages/girara.rb:16]: These 2 dependencies are included in crew "core" so not needed here.

File: packages/zathura.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  source_url 'https://pwmt.org/projects/zathura/download/zathura-0.4.5.tar.xz'
+  source_sha256 '0c3997aafbcdaaae60a4522f208adadfdd2758b432ce94ea16fbcee937cb762c'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  depends_on 'meson' => :build
+  depends_on 'gettext' => :build
```
Comment (uberhacker) [inline packages/zathura.rb:16]: These 2 dependencies are included in crew "core" so not needed here.

File: packages/zathura.rb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    system "ninja -C builddir"
+  end
+
+  def self.install
+    system "DESTDIR=#{CREW_DEST_DIR} ninja -C builddir install"
+  end
+
+  def self.postinstall
+    puts
+    puts "Please add plugins such as : zathura_poppler_pdf".lightblue
```
Comment (uberhacker) [inline packages/zathura.rb:30]: Maybe add the specific command to add a plugin for clarity?

File: packages/zathura_poppler_pdf.rb [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  source_url 'https://pwmt.org/projects/zathura-pdf-poppler/download/zathura-pdf-poppler-0.3.0.tar.xz'
+  source_sha256 'd5cc3a7dae49471b85b503bbb9049c6f8d10903f4a611d721a2e0fefe726d4ed'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  depends_on 'meson' => :build
+  depends_on 'gettext' => :build
```
Comment (uberhacker) [inline packages/zathura_poppler_pdf.rb:16]: These 2 dependencies are included in crew "core" so not needed here.

**Comment by uberhacker** [comment]
Thank you for the new packages @vincowl!  Please see comments.

**Comment by vincowl** [comment]
Sadly `#{CREW_OPTIONS}` doesn't work with `meson` due to the `--build` argument.

**Comment by uberhacker** [comment]
Yeah, that is expected.  `#{CREW_OPTIONS}` only works for `./configure`, `make` and `make install` source packages.  We could add a `#{CREW_NINJA_OPTIONS}` constant, however.  At a minimum, this could be `--prefix=#{CREW_PREFIX} --root=#{CREW_DEST_DIR}`.

**Pull request merged**